src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -23,8 +23,10 @@ class Lambda {
       (event.itemType, event.eventType) match {
         case (ItemType.Content, EventType.Delete) =>
           sendPurgeRequest(event.payloadId)
+        case (ItemType.Content, EventType.Update) =>
+          sendSoftPurgeRequest(event.payloadId)
         case other =>
-          // for now we only send purges for content takedowns, so ignore any other events
+          // for now we only send purges for content, so ignore any other events
```

Comment:
@LATaylor-guardian off the top of my head without thinking it through too much, things like an event on a tag when it is added to a piece of content. Things like RSS feeds are quite expensive and on a lot of our rss feeds (and likewise tag pages) we could have long cache times.